src/scripts.js

```diff
@@ -92,15 +135,26 @@ function showUsers() {
 function showRecipes() {
   let recipeHTML = '';
   recipeInstantiation.forEach(recipe => {
-    let recipeNames = recipe.getIngredients();
-    let showInstructions = recipe.getInstructions();
+    const recipeNames = recipe.getIngredients();
+    const showInstructions = recipe.getInstructions();
     let instructionsHTML = showInstructions.reduce((instructionDetail, instruction) => {
       instructionDetail += `<li>${instruction}</li>`;
       return instructionDetail;
     }, '')
-    let ingredientHTML = recipeNames.reduce((ingredientDetail, ingredient) => {
-      ingredientDetail += `<li>${ingredient}</li>`;
+    const amounts = recipe.ingredients.map(item => {
```

Comment:
Sweet, yeah, I think we can pull these methods out and put them in their own functions to dry this up!